Resources/Audio/StationEvents/attribution.txt

```diff
@@ -1 +1,3 @@
 countdown.ogg is created by qwertyquerty and licensed under CC-BY-SA-3.0. It is taken from https://github.com/BeeStation/BeeStation-Hornet at commit 79b8cc23cfb347cf23ea70fc5f6f39afedf9cde7.
+
+RunningOut is created by Bolgarich and licensed under CC-BY It is taken from https://www.youtube.com/watch?v=SzEp2nv6oZ4
```

Comment:
need to change this to say CC-BY-3.0 (see: the license under the youtube video) and provide a link to it - here's a link to it: https://creativecommons.org/licenses/by/3.0/legalcode

Reply:
Should be in attributions.yml, .txt is deprecated.